packages/pn-pa-webapp/src/redux/apiKeys/reducers.ts

```diff
@@ -13,14 +24,28 @@ const apiKeysSlice = createSlice({
   initialState,
   reducers: {
     resetState: () => initialState,
+    setPagination: (state, action: PayloadAction<{ page: number; size: number }>) => {
+      if (state.pagination.size !== action.payload.size) {
+        // reset pagination
+        state.pagination.nextPagesKey = [];
```

Comment:
here we will reset only the array of object `{lastKey: ..., lastUpdate: ...}`

Reply:
changes made into 5eb41f24bacd7862f6c9ac99ae9f9e9396e71a40